src/Control/Parallel.purs

```diff
@@ -48,3 +50,13 @@ parSequence_
   => t (m a)
   -> m Unit
 parSequence_ = parTraverse_ id
+
+parOneOf
+  :: forall a t m f
+   . Parallel f m
+  => Plus f
```

Comment:
Is it useful to have `Alternative` here, since we already have `Applicative` anyway? Seems like the additional laws might be nice to have.